src/requests/hover.jl

```diff
@@ -66,12 +66,33 @@ function get_hover(b::StaticLint.Binding, documentation::String, server)
     return documentation
 end
 
+# print(io, x::SymStore) methods are defined in SymbolServer
 function get_hover(b::SymbolServer.SymStore, documentation::String, server)
     if !isempty(b.doc)
         documentation = string(documentation, b.doc, "\n")
     end
     documentation = string(documentation, "```julia\n", b, "\n```")
-    # print(io, x::SymStore) methods are defined in SymbolServer
+end
+
+const JULIA_DIR = normpath(joinpath(Sys.BINDIR, Base.DATAROOTDIR, "julia"))
```

Comment:
We'll have to think hard about this one here because I think it might make the LS code non-relocatable, which will cause trouble when we want to statically compile it.

Maybe an easier way would be that we fix these paths in the symbol server process? That will always run in a Julia instance that "makes sense". But I'm not sure that works... @ZacLN?

Reply:
We can just set this at package load time, not at precompilation time (which we don't do at the moment anyways).

Reply:
I think that will still be wrong: say we statically compile LS with Julia 1.4.2, but the user has VS Code configured to use Julia 1.0.5. So in this case the LS process will run in Julia 1.4.2, but we want paths to the local install of Julia 1.0.5 of the user, which we won't get in the LS process. But we _would_ get those correct paths in the symbol server process because that would run under Julia 1.0.5.

Reply:
Is there an issue with the paths from SymbolServer other than that we don't want to display them here?

Reply:
I think (but not sure) the goal here is simply to return absolute paths for the files from base? I think we should easily be able to make that transformation in the symbol server.

Reply:
yes, I basically tried to mimic how `Base` displays methods:
```julia
julia> methods(sin)
# 14 methods for generic function "sin":
[1] sin(x::BigFloat) in Base.MPFR at mpfr.jl:727
[2] sin(a::Complex{Float16}) in Base.Math at math.jl:1146
[3] sin(a::Float16) in Base.Math at math.jl:1145
[4] sin(::Missing) in Base.Math at math.jl:1198
...
```

> I think we should easily be able to make that transformation in the symbol server.

We need absolute file path here since we want to create links in this package, so I guess the easiest workaround is just to drop this feature ? Or defining `JULIA_DIR` in SymbolServer.jl and use that here, maybe ?

Reply:
Yes, I think the latter: `JULIA_DIR` in the symbol server process will point to the correct location, even once we move the LS over to a statically compiled version.

Reply:
This line can be delete now, right?